services/vs-marketplace/vs-marketplace-version.service.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const VsMarketplaceBase = require('./vs-marketplace-base')
+const { addv } = require('../../lib/text-formatters')
+const { version: versionColor } = require('../../lib/color-formatters')
+
+module.exports = class VsMarketplaceVersion extends VsMarketplaceBase {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: '',
+      pattern: '(vscode-marketplace|vs-marketplace)/v/:extensionId',
+    }
+  }
+
+  static render({ version }) {
```

Comment:
You could use `renderVersionBadge` here.

Reply:
👍